File: Java/Processor.java
class Processor {
    
    private String name;
    private int price;

    Processor(){ }
    Processor(String name, int price){
        this.name = name;
        this.price = price;
    }

    void setNama(String name){
        this.name = name;
    }
    String getNama(){
        return this.name;
    }

    void setPrice(int price){
        this.price = price;
    }
    int getPrice(){
        return this.price;
    }

    void printDetProcee(){
        System.out.println("----------------------");
        System.out.println("Processor Name : " + this.name);
        System.out.println("Price          : " + this.price);
    }
}

File: Python/Processor.py
class Processor:

    def __init__(self, nama, price):
        self.nama = nama
        self.price = price

    def setNama(self, nama):
        self.nama = nama
    def getNama(self):
        return self.nama

    def setPrice(self, price):
        self.price = price
    def getPrice(self):
        return self.price

    def printDetProcee(self):
        print("----------------------")
        print("Processor Name : ", self.nama)
        print("Price          : ", self.price)

